### PR TITLE
Removes toxin healing virus symptom

### PR DESCRIPTION
**Pull request by Shadowlight213**
A long time coming.

:cl:
del: Removed the Toxic filter and Apoptoxin filter symptoms
/:cl:

By removing the toxic filter, the toxic comp symptom will not be a perfect heal virus. you'll still need to manage your toxin levels when you get hurt.

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -1,89 +1,6 @@
...
-/datum/symptom/heal
-
-	name = "Toxic Filter"
-	stealth = 1
-	resistance = -4
-	stage_speed = -4
-	transmittable = -4
-	level = 6
-
-/datum/symptom/heal/Activate(datum/disease/advance/A)
```
Comment (ChangelingRain) [inline code/datums/diseases/advance/symptoms/heal.dm]: You do still need this

**Review by lzimann — APPROVED** [review]
it has been long overdue..

**Comment by kevinz000** [comment]
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO :-1: :-1: :-1:

**Comment by PKPenguin321** [comment]
at most just nerf it

**Comment by drynwyn** [comment]
This basically removes the purpose of virology other than fucking over the station. There's no reason for a non-antag to do virology without these symptoms.

**Comment by PKPenguin321** [comment]
Well there's still like stimulants

**Comment by XDTM** [comment]
Without toxin filter, toxic compensation becomes just a liability due to toxin damage being harder to heal  and making you vomit. If you want to go down this path you should, instead, remove toxin compensation and have a different symptom for each kind of damage, so they're harder to stack.

**Comment by Ergovisavi** [comment]
Just fucking remove stimulants while you're at it as well please. LOL PERMANENT ANTI-STUN SPEEDBOOST.

**Comment by TechnoAlchemisto** [comment]
like most things in this game, virology is usually fine but some people abuse it to an extent where it has to be nerfed















@kevinz000

**Comment by chickenboy10** [comment]
Sure, I may be really good at virology, able to set up a super virus in under 30 minutes using just five sheets of uranium and three sheets of gold.
Sure, I may make 50 pills and throw them at everyone like in a bid to get everyone in the crew super heal, just so my work is appreciated.
But now you are going to nerf what seperates the regular "meh" virus from an actually "good" virus by nerfing virology this hard? Even though it costs rare materials to get the really good virus, as well as time and a bit of luck?
What a PR, literally only made because one round where I happened to be viro and bothered making my virus for the crew.

**Comment by lzimann** [comment]
how about we just remove viro already :^)

**Comment by ChemicalRascal** [comment]
Honestly, this was never a problem before Apoptoxin became a thing. Just remove that.

**Comment by jaxy15** [comment]
:+1: Why was legion soul regen removed but not toxic filter?

**Comment by Shadowlight213** [comment]
>All these people crying "VIRO IS WORTHLESS NOW REEEEE"
The only thing that's actually gone is the BS healing.

Maybe I should take people's advice and remove advanced symptoms entirely.

@XDTM I'd possibly consider removing toxic comp instead, but I'm definitely NOT adding symptoms to replace it. Hell, maybe I should remove both.

I remember at one point I had an idea to maybe make the heal viruses not work in crit or ultra low health. But no, this system of infinite regeneration is bullshit.

**Comment by ChemicalRascal** [comment]
_This_ system, or _all_ systems of infinite regeneration?

**Comment by Shadowlight213** [comment]
@ChemicalRascal just viro

**Comment by ChemicalRascal** [comment]
@Shadowlight213 I don't really understand why you view viro as different to other systems, then. If your argument was that it's currently too easy and too quick, I'd agree with that -- it really should be a lategame thing -- but by removing viro regen wholesale, you're removing one of the very few ways that non-antags have to at least partially mitigate antag shenanigans.

A viro super-healing virus thirty minutes in, sure, that probably shouldn't be a thing, but otherwise a healing virus simply means that antags need to burst their damage more effectively, they can't beat people to death with a bike horn. And it's not a problem for non-antags to influence antag playstyle in such a minor way, because it is so incredibly minor. In some ways, it's kind of a good thing.

It'd be better to use this as an opportunity to make viro complicated and introduce important, non-trivial decisions to the process, instead of just mashing the delete button, and removing all non-antag reasons to play viro.

**Comment by Shadowlight213** [comment]
@ChemicalRascal What other infinite nonstop full regen systems are there? Other than xenos and ayy glands, I can't think of any.
Also, medbay is for when you get hurt as a non antag, not your own personal heal system.
Antags don't have the luxury of asking others to help once they've been outed.

**Comment by ChemicalRascal** [comment]
@Shadowlight213 You're not reading what I'm writing, dude.

It's one of the very few partial mitigation methods general non-antags have available to them. It's _overpowered_ in its current form, given that those level eight symptoms can be hit relatively quickly, but removing viro healing entirely will only tip things in the direction of antags even further than they currently are.

**Comment by KorPhaeron** [comment]
Please do this properly. Removing this symptom leaves behind a useless room on every map and a useless job.

Either replace the symptom with something else or go all the way and remove viro.

**Comment by Shadowlight213** [comment]
@KorPhaeron you're acting like I'm removing EVERY SINGLE BENEFICIAL SYMPTOM.
I'm not. There's still sensory restoration, self resp, dna restoration, etc..
Do those not exist or something? Is regen the only beneficial symptom that exists??
I can remove viro all the way. Sure.

**Comment by Kraseo** [comment]
Why don't we remove all forms of self healing? Actually, let's remove all of medbay as a whole.

**Comment by KorPhaeron** [comment]
>repairing eye damage 
>worth an entire room and job slot

No

**Comment by nullbear** [comment]
i agree with nerfing it. Removing it is dumb. Just make it heal much slower?

**Pull request closed without merging**